Sming/Components/rboot/component.mk

```diff
@@ -39,19 +39,22 @@ endif
 
 CONFIG_VARS				+= RBOOT_ROM0_ADDR RBOOT_ROM1_ADDR RBOOT_ROM2_ADDR
 
-# Loation of first ROM (default is sector 2 after rboot and rboot config sector)
+# Location and size of first ROM (default is sector 2 after rboot and rboot config sector)
 RBOOT_ROM0_ADDR			?= 0x002000
+RBOOT_ROM0_SIZE			?= (0x100000 - $(RBOOT_ROM0_ADDR))
```

Comment:
Note: `RBOOT_ROMx_ADDR` may be larger than 0x100000 (for devices with more than 1MB of flash memory). Hence this calculation yields a negative value.